.github/workflows/zeebe-benchmark.yml

```diff
@@ -119,10 +109,22 @@ jobs:
           vault-address: ${{ secrets.VAULT_ADDR }}
           vault-role-id: ${{ secrets.VAULT_ROLE_ID }}
           vault-secret-id: ${{ secrets.VAULT_SECRET_ID }}
+      - uses: ./.github/actions/build-frontend
```

Comment:
:thought_balloon: Surprised there's not a global `build-frontend` action :shrug:

Reply:
That would be too easy.